tests/installation/partitioning_lvm.pm

```diff
@@ -86,38 +86,43 @@ sub run {
             $collect_logs = 1;
         }
     }
-    elsif (is_storage_ng) {
-        if ($numdisks <= 1) {
-            die "Guided workflow does not skip disk selection when only one disk!"
-              unless match_has_tag('inst-partitioning-scheme');
-        }
-        else {
-            die "Guided workflow does skip disk selection when more than one disk! (NUMDISKS=$numdisks)"
-              unless match_has_tag('inst-partition-guided');
-            assert_screen "inst-partition-guided";
-            send_key 'alt-n';
-            assert_screen "inst-select-root-disk";
-            send_key 'alt-n';
-        }
-        send_key $cmd{enablelvm};
-        assert_screen "inst-partitioning-lvm-enabled";
-        if (get_var("ENCRYPT")) {
-            enable_encryption_guided_setup;
-        }
-        else {
+    if (match_has_tag('inst-select-disk-to-use-as-root')) {
+        if (is_storage_ng) {
+            if ($numdisks <= 1) {
+                die "Guided workflow does not skip disk selection when only one disk!"
+                  unless match_has_tag('inst-select-disk-to-use-as-root');
```

Comment:
we should fail unconditionally here, as match_has_tag('inst-select-disk-to-use-as-root') is already above